google_compute_engine/compat.py

```diff
@@ -38,6 +38,10 @@
   import google_compute_engine.distro.debian_8.utils as distro_utils
 elif 'debian' in distro_name and distro_version == '9':
   import google_compute_engine.distro.debian_9.utils as distro_utils
+elif 'suse' in distro_name and distro_version == '11':
+  import google_compute_engine.distro.sles_11.utils as distro_utils
+elif 'suse' in distro_name and distro_version == '12':
```

Comment:
It might be better to remove the `and distro_version == '12'`. That way other suse versions will run code that has some chance of working.

Reply:
agreed, without version test for 12, openSUSE Leap 42.3 will work as well as the upcoming versions openSUSE Leap 15 and SLE 15.